resources/migrations/052_add_pulse_skip_if_empty.yaml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: 52
```

Comment:
Sorry to be a PITA but I have migration 52 and 53 in the `caching` branch so could you rename this file to `054_add_pulse_skip_if_empty.yaml` and change the id to `54`? Otherwise if I renumber the migrations in the caching branch it more people will have to manually fix their migration histories since more people have been playing with that branch